include/VulkanInfoProvider.inl

```diff
@@ -0,0 +1,3 @@
+inline const nlohmann::json& VulkanInfoProvider::getJSONVulkanInfo() const {
```

Comment:
I think that this file is not used anymore, it's not included in `VulkanInfoProvider.hpp` and the property `_json` doesn't exist.